tests/ut_seattlelibv2_repypp_include_noargument.py

```diff
@@ -0,0 +1,30 @@
+#pragma out Error opening source file ''
+"""
+repypp.py is a preprocessor for repy. It includes dependent files as needed.This is used to help the 
+programmer avoid the need to use import. They can instead use "include X" which works somewhat like "from X import *".
+
+This script tests if repypp.py check erroneous #include statements(have no arguments at all).
+"""
+
+import os
+import sys
+import subprocess
+
+def main():
+    #create a temporary_file2 contains:
+    #include 
+    #def bar():
+    #  pass
+    temporary_file = open('testfile_repypp_example.repy', 'w')
+    temporary_file.write('include  \ndef bar():\n  pass\n') 
```

Comment:
Nitpicking: Is the error identical for different amounts of trailing whitespace, i.e. "`include  `", "`include `", and "`include`"?